__mocks__/inquirer/prompt.mock.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
```

Comment:
Why is ESLint was disabled for this file? If you want to disable specific rules, please be specific about it and disable only those.